docs/core/versions/remove-runtime-sdk-versions.md

```diff
@@ -196,3 +196,21 @@ sudo rm -rf /usr/local/share/dotnet/host/fxr/1.0.1
 The parent directories for the SDK and runtime are listed in the output from the `dotnet --list-sdks` and `dotnet --list-runtimes` command, as shown in the earlier table.
 
 ---
+
+## .NET Core Uninstall Tool
+
+The [.NET Core Uninstall Tool (`dotnet-core-uninstall`)](https://dotnet.microsoft.com/download/dotnet-core/uninstall-tool) is a guided tool provided to enable the controlled clean-up of a system such that only the desired versions of .NET Core SDKs and Runtimes remain. A collection of options is available to facilitate the aggregation of uninstallations.
+
+The documentation of the tool is available [here](../additional-tools/dotnet-core-uninstall.md).
+
+## Visual Studio dependencies on .NET Core SDK versions
+
+Visual Studio installers prior to Visual Studio 2019 16.3 install a standalone .NET Core SDK, which is listed in the Windows **Add/Remove Programs** dialog. Hence, be aware that removing .NET Core SDKs installed via the Visual Studio installer prior to Visual Studio 2019 16.3 may break the Visual Studio. The following table shows some of the Visual Studio dependencies on .NET Core SDK versions.
```

Comment:
```suggestion
Visual Studio installers prior to Visual Studio 2019 16.3 called the standalone .NET Core SDK installer. As a result, these versions of the SDK appear in the Windows **Add/Remove Programs** dialog. Hence, be aware that removing .NET Core SDKs installed prior to Visual Studio 2019 16.3 may break Visual Studio. If Visual Studio has problems after you uninstall SDKs, run Repair on that version of Visual Studio. The following table shows some of the Visual Studio dependencies on .NET Core SDK versions:
```

Reply:
"called" or "calls"?

I'm not sure on this.